heartbeat/docs/heartbeat-observer-options.asciidoc

```diff
@@ -1,7 +1,28 @@
 [[configuration-observer-options]]
 == Add observer and geo metadata
 
-It is often useful to view and query various attributes of the instance Heartbeat is running on. This data is attached to events via the <<add-observer-metadata,`add_observer_metadata`>> processor. This processor populates the ECS `observer.*` fields. One field of particular utility is the `observer.geo.name` field, which you can configure via this processor. Use this field to create distinctive geographic regions for your uptime checks. You might label your regions by datacenter name or geographic region, e.g. `virginia-dc-1`, or `us-east-1a`, or `virginia-us`.
+Use the `heartbeat.run_from` option to set the geographic location fields relevant to a given heartbeat instance.
 
-Note that Heartbeat should not be used with the similarly named <<add-host-metadata,`add_host_metadata`>> processor. In ECS parlance the host is the thing that is monitored, not the thing doing the monitoring, which is the observer.
+Determines which geographic location to label this monitor as running from. Note, this
+can also be handled by setting the `run_from` option on an individual monitor to apply
+a unique setting to just that monitor.
```

Comment:
A couple thoughts:

* The first sentence here is unclear. What is doing the determining?
* The second sentence could be simplified.
* Is something like the suggestion below still accurate? Feel free to adjust as needed.

```suggestion
The `run_from` option is used to label the geographic location where the monitor is running.
Note, you can also set the `run_from` option on an individual monitor to apply a unique setting to just that monitor.
```